content/redirects.txt

```diff
@@ -919,18 +919,15 @@
 /docs/providers                     /docs/language/providers/index.html
 
 # Clean up Extend section (now Plugin Development)
-#/docs/extend/community/code-of-conduct.html    https://www.hashicorp.com/community-guidelines
-#/docs/extend/community/contributing.html       https://github.com/hashicorp/terraform/blob/main/.github/CONTRIBUTING.md
-#/docs/extend/community/events/index.html       https://www.terraform.io/community
-#/docs/extend/community/index.html              https://www.terraform.io/community
-#/docs/extend/community                         https://www.terraform.io/community
-#/docs/extend/community/                        https://www.terraform.io/community
-#/docs/extend/community/maintainers.html        https://github.com/hashicorp/terraform/blob/main/.github/CONTRIBUTING.md#maintainers
-/docs/extend/plugin-types.html                  /docs/extend/index.html
+/docs/extend/community/events/index.html                  /community
+/docs/extend/community/events/2018/fall-gardening.html    /community
+/docs/extend/community/index.html                         /community
+/docs/extend/community                                    /community
+/docs/extend/community/                                   /community
```

Comment:
`https://www.terraform.io/community` and `/community` _should_ lead to the same place, so I thought it best to make these internal. If they were meant to be considered external, I'm willing to change them back!